File: gh-2516/src/main/java/com/example/gh2516/Wallet.java
package com.example.gh2516;

import org.springframework.data.neo4j.core.schema.GeneratedValue;
import org.springframework.data.neo4j.core.schema.Id;
import org.springframework.data.neo4j.core.schema.Node;

@Node
public class Wallet {

	@Id
	@GeneratedValue
	public Long id;

	private String name;
}

File: gh-2516-kt/src/main/kotlin/com/example/gh2516kt/Wallet.kt
package com.example.gh2516kt

import org.springframework.data.neo4j.core.schema.GeneratedValue
import org.springframework.data.neo4j.core.schema.Id
import org.springframework.data.neo4j.core.schema.Node

@Node
class Wallet {
    @Id
    @GeneratedValue
    var id: Long? = null
    private val name: String? = null
}

